example/cypress/support/commands.js

```diff
@@ -46,4 +46,18 @@ Cypress.Commands.add('refreshDatabase', () => {
             }
         });
     });
+});
+
+Cypress.Commands.add('createUser', (attributes) => {
+    return cy.csrfToken().then((token) => {
+        return cy.request({
+            method: 'POST',
+            url: '/__cypress__/createUser/',
+            body: { ...attributes  },
```

Comment:
Spreading `attributes` is not required, right? What do you think?

Reply:
Correct, fixed it 😄

Reply:
Thanks!!!